packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -718,10 +713,6 @@ class _SerializeConfig {
   /// How many levels of children to include in the JSON payload.
   final int subtreeDepth;
 
-  /// Path of nodes through the children of this node to include even if
-  /// subtreeDepth is exceeded.
-  final Iterable<Diagnosticable> pathToInclude;
```

Comment:
This property is only set in a branch `if (pathToInclude != null)`, see below. Since it is never set to anything outside of that branch, that branch can never activate.